src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* Addressed TypeDescriptor.GetProperties warning by removing it.
+  ([#4560](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4560))
+  **Side effect:** Skip processing log state if the state neither implements
+  IReadOnlyList nor IEnumberable. This is a breaking change that removed the
+  support introduced in PR4334 that used `TypeDescriptor.GetProperties()` to add
```

Comment:
"the supported introduced in PR4334" doesn't seem to be how users would be able to map in their mind - I guess the user would care about "feature XYZ introduced in version ABC is not being removed", a user shouldn't have to learn about a PR and map it to a release version